Formula/wine.rb

```diff
@@ -37,11 +37,11 @@ def require_universal_deps?
   depends_on "pkg-config" => :build
   depends_on "freetype"
   depends_on "jpeg"
-  depends_on "libgphoto2"
+  depends_on "libgphoto2" => :optional
```

Comment:
I'd probably make these `:recommended` for now. We try not to suddenly downgrade dependencies by default between versions in case someone's relying on the status quo.